internal/api/entry.go

```diff
@@ -279,7 +279,11 @@ func (h *handler) fetchContent(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	json.OK(w, r, map[string]string{"content": entry.Content})
+	if err := h.store.UpdateEntryContent(entry); err != nil {
```

Comment:
That certainly could be changed, I was considering adding an additional endpoint that would save it and return it, and leave that endpoint alone.
The use-case for saving it is inter-op with a mobile application, and keeping that in sync with the desktop version.

If you would prefer, I can remove that change.

Reply:
I was also considering changing the behaviour of the `download` button to not refresh the content if it is already saved, and adding a button to refresh. I didn't do this because it didn't really seem necessary yet.

Reply:
hmm, yeah optional query parameter is a good idea

Reply:
I think this can probably solve it?
```go
if request.QueryBoolParam(r, "save", false) {
  if err := h.store.UpdateEntryContent(entry); err != nil {
    json.ServerError(w, r, err)
  }
}
```